azure-pipelines.yml

```diff
@@ -29,30 +31,121 @@ stages:
+        _TargetFramework: 'netcoreapp3.1'
+        _ToolPlatform: x86
+      Debug-AnyCPU-netcoreapp5.0:
+        _Configuration: Debug
+        _Platform: 'Any CPU'
+        _TargetFramework: 'netcoreapp5.0'
+        _ToolPlatform: x86
+      Debug-x64-netcoreapp3.0:
+        _Configuration: Debug
+        _Platform: 'x64'
+        _TargetFramework: 'netcoreapp3.0'
+        _ToolPlatform: x64
+      Debug-x64-netcoreapp3.1:
+        _Configuration: Debug
+        _Platform: 'x64'
+        _TargetFramework: 'netcoreapp3.1'
+        _ToolPlatform: x64
+      Debug-x64-netcoreapp5.0:
+        _Configuration: Debug
+        _Platform: 'x64'
+        _TargetFramework: 'netcoreapp5.0'
+        _ToolPlatform: x64
+      Debug-x86-netcoreapp3.0:
         _Configuration: Debug
-        _Platform: x64
-      Build_Release_x64:
+        _Platform: 'x86'
+        _TargetFramework: 'netcoreapp3.0'
+        _ToolPlatform: x86
+      Debug-x86-netcoreapp3.1:
+        _Configuration: Debug
+        _Platform: 'x86'
+        _TargetFramework: 'netcoreapp3.1'
+        _ToolPlatform: x86
+      Debug-x86-netcoreapp5.0:
+        _Configuration: Debug
+        _Platform: 'x86'
+        _TargetFramework: 'netcoreapp5.0'
+        _ToolPlatform: x86
+      Release-AnyCPU-netcoreapp3.0:
+        _Configuration: Release
+        _Platform: 'Any CPU'
+        _TargetFramework: 'netcoreapp3.0'
+        _ToolPlatform: x86
+      Release-AnyCPU-netcoreapp3.1:
+        _Configuration: Release
+        _Platform: 'Any CPU'
+        _TargetFramework: 'netcoreapp3.1'
+        _ToolPlatform: x86
+      Release-AnyCPU-netcoreapp5.0:
+        _Configuration: Release
+        _Platform: 'Any CPU'
+        _TargetFramework: 'netcoreapp5.0'
+        _ToolPlatform: x86
+      Release-x64-netcoreapp3.0:
+        _Configuration: Release
+        _Platform: 'x64'
+        _TargetFramework: 'netcoreapp3.0'
+        _ToolPlatform: x64
+      Release-x64-netcoreapp3.1:
         _Configuration: Release
-        _Platform: x64
+        _Platform: 'x64'
+        _TargetFramework: 'netcoreapp3.1'
+        _ToolPlatform: x64
+      Release-x64-netcoreapp5.0:
+        _Configuration: Release
+        _Platform: 'x64'
+        _TargetFramework: 'netcoreapp5.0'
+        _ToolPlatform: x64
+      Release-x86-netcoreapp3.0:
+        _Configuration: Release
+        _Platform: 'x86'
+        _TargetFramework: 'netcoreapp3.0'
+        _ToolPlatform: x86
+      Release-x86-netcoreapp3.1:
+        _Configuration: Release
+        _Platform: 'x86'
+        _TargetFramework: 'netcoreapp3.1'
+        _ToolPlatform: x86
+      Release-x86-netcoreapp5.0:
+        _Configuration: Release
+        _Platform: 'x86'
+        _TargetFramework: 'netcoreapp5.0'
+        _ToolPlatform: x86
     steps:
      - task: NuGetToolInstaller@1
        displayName: 'Download and Cache Nuget tool'
 
+     - powershell: |
+         Copy-Item .\eng\$(_TargetFramework).json global.json -Force -Verbose
```

Comment:
Is this used anymore? This file doesn't exist in the branch and doesn't ensure create the json on the fly?

Reply:
Right - good catch!